qna/carefree-extension-after-expired-grace-period.md

```diff
@@ -0,0 +1,15 @@
+## Question
+
+How to extend Carefree products which expired beyond the grace period?
+
+## Metadata tags
+
+lang-en, market-at, bundle-carefree, portal, cashbox management, PosOperator, PosDealer, Consultant, support-required
```

Comment:
Tag name "support-required" is not part of the tag list (see landing page of the repo). Can you please remove it

Reply:
The file-name should be how described in the landing page with starting eg "AT-":
"AT-carefree-extension-after-expired-grace-period.md"

Reply:
> The file-name should be how described in the landing page with starting eg "AT-":
> "AT-carefree-extension-after-expired-grace-period.md"

The file has been renamed. Thanks for the suggestion!

Reply:
> Tag name "support-required" is not part of the tag list (see landing page of the repo). Can you please remove it

The tag has been removed. Thank you for spotting it!